src/Serokell/Data/Variant/Variant.hs

```diff
@@ -10,23 +10,18 @@ module Serokell.Data.Variant.Variant
        , VarMap
        ) where
 
-import           Control.DeepSeq      (NFData)
-import           Data.ByteString      (ByteString)
-import           Data.Hashable        (Hashable (hashWithSalt))
-import           Data.HashMap.Strict  (HashMap)
-import qualified Data.HashMap.Strict  as HM hiding (HashMap)
-import           Data.Int             (Int64)
-import           Data.String          (IsString (fromString))
-import           Data.Text            (Text)
-import           Data.Text.Buildable  (Buildable (build))
-import           Data.Vector          (Vector)
-import qualified Data.Vector          as V hiding (Vector)
-import           Data.Word            (Word64)
-import           GHC.Exts             (IsList (..))
-import           GHC.Generics         (Generic)
+import Universum
 
+import Control.DeepSeq (NFData)
+import Data.Hashable (Hashable (hashWithSalt))
```

Comment:
Recent `universum` exports `Hashable` and `hashWithSalt`. You even did this change 🙂

Reply:
I remember this, but it was done in later version of `universum` :slightly_smiling_face: